spec/rights_auth_spec.rb

```diff
@@ -54,8 +54,40 @@
     XML
   end
 
-  context 'controlled digital lending rights' do
+  describe 'stanford + world (no-download) for a file' do
```

Comment:
so this is what you implemented?   This failed before?   I can see how this would conflict with the previous test.  I would say alter the previous test to be clear that it is about multiple access blocks with location restriction.